src/Log/Channels/LogChannel.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Log\Channels;
+
+use Monolog\Handler\HandlerInterface;
+use Monolog\Level;
+use Monolog\Processor\ProcessorInterface;
+
+interface LogChannel
```

Comment:
this could potentially also just have a method that returns a LogInterface, I did it this way because it should be easier to implement stack / multiple channels support